rakefile.rb

```diff
@@ -47,8 +47,7 @@
     - [ ] attach nupkg
     - [ ] publish the release
 - [ ] push NuGet package
-- [ ] copy release notes from GitHub to NuGet
-- [ ] de-list pre-release or superseded buggy NuGet packages if present (copy any release notes forward to the new version)

```

Comment:
:scream: 

![image](https://cloud.githubusercontent.com/assets/677704/15447960/bd89394e-1f52-11e6-8082-8ce04ff50985.png)

I love it when a (micro-commit) plan comes together... :wink: 

I think it's mainly PR merges though. https://github.com/FakeItEasy/FakeItEasy/graphs/contributors doesn't take those into account.